seacrowd/sea_datasets/filipino_words_aoa/filipino_words_aoa.py

```diff
@@ -0,0 +1,131 @@
+_DATASETNAME = "filipino_words_aoa"
+_DESCRIPTION = """\
+The dataset contains 885 Filipino words derived from an age-of-acquisition participant study. The words are derived child-directed corpora
+using pre-specified linguistic criteria. Each word in the corpora contains information about its meaning, part-of-speech (POS), age band,
+morpheme count, syllable length, phoneme length, and the level of book it was derived from. The dataset can be used for lexical complexity
+prediction, lexical simplification, and readability assessment research.
+"""
+
+_HOMEPAGE = "https://osf.io/3zdfn/"
+_LICENSE = Licenses.CC_BY_SA_4_0.value
+_URL = "https://osf.io/download/j42g7/"
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class FilipinoWordsAOADataset(datasets.GeneratorBasedBuilder):
+    """\
+    Dataset of Filipino words, their English meanings, and their part-of-speech tag
+    obtained from an age-of-acquisition study.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_t2t",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SeaCrowd text-to-text schema",
+            schema="seacrowd_t2t",
+            subset_id=_DATASETNAME,
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "word": datasets.Value("string"),
+                    "meaning": datasets.Value("string"),
+                    "POS_tag": datasets.Value("string"),
+                    "mean_AoA": datasets.Value("float64"),
+                    "mean_AoA_ageband": datasets.Value("string"),
+                    "morpheme_count": datasets.Value("int64"),
+                    "syllable_length": datasets.Value("int64"),
+                    "phoneme_length": datasets.Value("int64"),
+                    "book_ageband": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "seacrowd_t2t":
+            features = schemas.text2text_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        filepath, _ = urllib.request.urlretrieve(_URL)
```

Comment:
Is there any reason why we're not using the `dl_manager`? 

```suggestion
        filepath = dl_manager.download(_URL)
```

Try this one first and see if your code still runs (I check locally and it did). Remember to remove the `urllib` import above.

Reply:
Wasn't just familiar with `dl_manager` in general, thanks for letting me know about this!